packages/javascript-sdk/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 4.8.1
+
+### Patch Changes
+
+- [#544](https://github.com/ForgeRock/forgerock-javascript-sdk/pull/544) [`58360de`](https://github.com/ForgeRock/forgerock-javascript-sdk/commit/58360de68ed248961cb9b7c6433594faa781c7ec) Thanks [@ryanbas21](https://github.com/ryanbas21)! - Fix condition for determining session endpoint request for terminating session in AIC/AM
```

Comment:
```suggestion
- [#544](https://github.com/ForgeRock/forgerock-javascript-sdk/pull/544) [`58360de`](https://github.com/ForgeRock/forgerock-javascript-sdk/commit/58360de68ed248961cb9b7c6433594faa781c7ec) Thanks [@ryanbas21](https://github.com/ryanbas21)! - Fixed how the SDK determines the endpoint to use for terminating a session in AIC and PingAM
```

Reply:
Be helpful to me if we could include the Jira ID as a prefix to each changeset entry.